### DISCUSSION TITLE
SSL on alias domains (WP Multisite Subdom - additional domains)

### DISCUSSION THREAD

**Discussion by OctavioSI** [EasyEngine/easyengine#1835] [Support Requests]
Hi everyone -- I looked back and forth but unfortunately I was not able do find an answer for that.

**TL;DR;
How do I validate SSL certificate when adding a new alias domain (new domain, not subdom)?**

*The problem below:*

I've create a site on my main domain as such:
```
ee site create mycurrentsite.com --wpsubdom
```

After the site creation, I've updated it to use ssl and wildcards, as such:
```
ee site update mycurrentsite.com --ssl=le --wildcard
```
And everything works fine: I have my main domain with SSL and I am able to create new subdom sites within Wordpress interface. 

I am trying to now update my site with additional alias domains so that I can point such subdom sites to such additional domains instead. Changing the main address in Wordpress works fine, but obviously there is no SSL certificate for such additional domain.

Therefore, I am trying to include my additional domain using:
```
ee site update mycurrentsite.com --add-alias-domains='additionaldomain.com,*.additionaldomain.com'
```
This command seems to update the certificate issued to my main domain so as to also include the additional domains, but it asks me to update my DNS registers to include a TXT record to the additional domain (_acme-challenge). 

I've created such registers, but how can I ssl-verify this? Since it is an additional domain and not a new site, I cannot do a ```ee site ssl-verify additionaldomain.com``` on the additional domain, and if I try to rerun the add-alias-domains command, I am asked once again to create such TXT records.

Any ideas? Thank you all in advance.

For reference, the output I mentioned when trying to add the alias domain is the following:
```
ee site update mycurrentsite.com --add-alias-domains='additionaldomain.com,*.additionaldomain.com'
Updating and force renewing SSL certificate to accomodated alias domain changes.
Starting SSL cert renewal
Loading current certificate for mycurrentsite.com
    Add the following TXT record to your DNS zone
        Domain: _acme-challenge.additionaldomain.com.
        TXT value: 5BdNTtW_L8VqKJVnTkPBmP_qug0RmzTpOwxQQreKh2s
        
    Wait for the propagation before moving to the next step
    Tips: Use the following command to check the propagation

        host -t TXT _acme-challenge.additionaldomain.com.

    Add the following TXT record to your DNS zone
        Domain: _acme-challenge.additionaldomain.com.
        TXT value: FwKed_rwNEogfHIrwlTkvpCHBm6T4TVjKw6w_PcodDU
        
    Wait for the propagation before moving to the next step
    Tips: Use the following command to check the propagation

        host -t TXT _acme-challenge.additionaldomain.com.

    Add the following TXT record to your DNS zone
        Domain: _acme-challenge.mycurrentsite.com.
        TXT value: korirpfXly9qC66PnklufM8lwUGRLDxegbhNngpMe-o
        
    Wait for the propagation before moving to the next step
    Tips: Use the following command to check the propagation

        host -t TXT _acme-challenge.mycurrentsite.com.

IMPORTANT: Run `ee site ssl-verify mycurrentsite.com` once the DNS changes have propagated to complete the certification generation and installation.Starting site's services.
Warning: Certificate could not be issued. Reverting back to original state.
Enabling site mycurrentsite.com.
Success: Site mycurrentsite.com enabled.
Running post enable configurations.
Starting site's services.
Success: Post enable configurations complete.
```
